compiler/modules/front.syntax/parseTypeDesc.bal

```diff
@@ -388,3 +402,18 @@ function parseRecordTypeDesc(Tokenizer tok, Position startPos) returns MappingTy
     check tok.advance();
     return { startPos, endPos, fields, rest };
 }
+
+function parseInclusiveRecordTypeDesc(Tokenizer tok, Position startPos) returns MappingTypeDesc|err:Syntax {
+    check tok.advance();
+    FieldDesc[] fields = [];
+    while tok.current() != "}" {
+        TypeDesc td = check parseTypeDesc(tok);
+        Position fieldStartPos = tok.currentStartPos();
```

Comment:
Make this share code with parseExclusiveRecordTypeDesc.

Reply:
factored out `parseFieldDesc`